src/directLine.ts

```diff
@@ -337,8 +338,12 @@ export class DirectLine implements IBotConnection {
             }
         }
 
-        if (options.pollingInterval !== undefined) {
-            this.pollingInterval = options.pollingInterval;
+        if (options.pollingInterval !== undefined && typeof options.pollingInterval === 'number') {
```

Comment:
I think only `typeof options.pollingInterval === 'number'` is enough here.

Reply:
ah, true